docs/release/release_0_5_0.md

```diff
@@ -0,0 +1,911 @@
+painting labels ([#5806](https://github.com/napari/napari/pull/5806)),
+pinch-to-zoom ([#5859](https://github.com/napari/napari/pull/5859)), better
+ways to show/hide individual layers when exploring your data
+([#5574](https://github.com/napari/napari/pull/5574))
+([#5618](https://github.com/napari/napari/pull/5618)), and more:
+Over 20 new features in all and over 100 bug fixes and improvements!
+Please see below for the full list of changes since 0.4.19.
+
+## New Features
+
+- Add layer slicer base class for async slicing ([napari/napari#5170](https://github.com/napari/napari/pull/5170))
+- Feature: add layer contextual menu to hide not-selected layers ([napari/napari#5574](https://github.com/napari/napari/pull/5574))
+- Feature: alt-click visibility eye to toggle show just that layer (hide others) ([napari/napari#5618](https://github.com/napari/napari/pull/5618))
+- Feature: Add action to set current label to background (with keybinding) ([napari/napari#5672](https://github.com/napari/napari/pull/5672))
+- Use Dims margin in layer slicing code ([napari/napari#5697](https://github.com/napari/napari/pull/5697))
+- Changing brush size with mouse movement while holding specified modifiers ([napari/napari#5753](https://github.com/napari/napari/pull/5753))
+- Feature: polygon drawing in the Labels layer ([napari/napari#5806](https://github.com/napari/napari/pull/5806))
+- Replace old async loading with new approach ([napari/napari#5816](https://github.com/napari/napari/pull/5816))
+- Enable pinch-to-zoom with vispy 0.13+ ([napari/napari#5859](https://github.com/napari/napari/pull/5859))
+- Keep shading light for Surface fixed relative to camera ([napari/napari#5893](https://github.com/napari/napari/pull/5893))
+- Use Request/Response pattern for Vectors layer ([napari/napari#5918](https://github.com/napari/napari/pull/5918))
+- Lock dimensions / axes while rolling ([napari/napari#5986](https://github.com/napari/napari/pull/5986))
+- Added support for features in surface layers ([napari/napari#6515](https://github.com/napari/napari/pull/6515))
+- Add creating image from clipboard ([napari/napari#6532](https://github.com/napari/napari/pull/6532))
+- Add viewer keybinds to select & show only next/prev layer and to toggle visibility of unselected ([napari/napari#6590](https://github.com/napari/napari/pull/6590))
+- Show `About napari` action on macOS application menu ([napari/napari#6666](https://github.com/napari/napari/pull/6666))
+- Add points/shapes highlight color setting to Appereance settings and preference page ([napari/napari#6689](https://github.com/napari/napari/pull/6689))
+- Include features index as key in FormatStringEncoding ([napari/napari#6703](https://github.com/napari/napari/pull/6703))
+- Reset all dims to be rollable ([napari/napari#6797](https://github.com/napari/napari/pull/6797))
+- implement copying spatial information via clipboard ([napari/napari#6864](https://github.com/napari/napari/pull/6864))
```

Comment:
This is also pretty sweet. Not sure how long we want the highlight section, but worth pointing to IMO.